code-studio/src/js/initApp/initApp.js

```diff
@@ -33,11 +33,42 @@ window.apps = {
 
     var lastSavedProgram;
 
+    var containedLevelOps;
+    if (appOptions.hasContainedLevels) {
+      containedLevelOps = {
+        registerAnswerChangedFn: function (answerChangedFn) {
+          if (window.levelGroup) {
```

Comment:
I realize this file already uses a lot of globals, but it would be great if we could import this vs. pulling it from `window`.